document/core/exec/instructions.rst

```diff
@@ -3682,65 +3682,71 @@ Memory Instructions
 
 9. Let :math:`\X{mem}_s` be the :ref:`memory instance <syntax-meminst>` :math:`S.\SMEMS[\X{sa}]`.
 
-10. Assert: due to :ref:`validation <valid-memory.copy>`, a value of :ref:`value type <syntax-valtype>` |I32| is on the top of the stack.
+10. Let :math:`\X{it}_d~\limits` be the :math:`\X{mem}_d.\MITYPE`.
 
-11. Pop the value :math:`\I32.\CONST~n` from the stack.
+11. Let :math:`\X{it}_s~\limits` be the :math:`\X{mem}_s.\MITYPE`.
 
-12. Assert: due to :ref:`validation <valid-memory.copy>`, a value of :ref:`value type <syntax-valtype>` |I32| is on the top of the stack.
+12. Let :math:`\X{it}_{min}` be ....?
```

Comment:
```suggestion
12. Let :math:`\X{it}_{\X{min}}` be ....?
```
And similarly in other places. Or perhaps name this one just plain "it"?

I would say "... be the :ref:`minimum <aux-idxtype-min>` of ...` and then define that as a convention in the syntax section for index types.

Reply:
Done.  Although I'm not sure how you would me to define `aux-idxtype-min` itself.   Right now I've described it in prose but I imagine it warrants some kind of formal notation.